pubspec.yaml

```
@@ -35,6 +35,8 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
+  firebase_core: ^2.16.0
+  firebase_auth: ^4.10.0
```

Provide package url and why it's been added. 

For example:
```suggestion
  firebase_core: ^2.16.0  # https://pub.dev/packages/firebase_core - it's required as it's been used by firebase_auth
  firebase_auth: ^4.10.0
```

---

Done